services/files/run_push.go

```diff
@@ -305,49 +305,52 @@ func getGitBranch() (string, error) {
 	return filepath.Base(strings.TrimSpace(string(head))), nil
 }
 
-func (s service) runJob(ctx context.Context, params PushParams, files []string, project string) error {
+func (s service) runPushWithJob(ctx context.Context, params PushParams, files []string, project string) error {
 	fileUris, err := getFileUris(s.Config.Path, params, files)
 	if err != nil {
 		return err
 	}
-	var jobName string
-	if params.JobUID != "" {
-		if err := uuid.Validate(params.JobUID); err != nil {
-			jobName = params.JobUID
-		}
+	// create new job if params.JobIDOrName is not a valid UUID
+	pattern := `^[a-z0-9]{12}$`
+	var jobID string
+	if re := regexp.MustCompile(pattern); re.MatchString(params.JobIDOrName) {
+		jobID = params.JobIDOrName
 	}
 	var createJobResponse sdkjobs.CreateJobResponse
-	if jobName != "" {
+	if jobID == "" {
 		payload := sdkjobs.CreateJobPayload{
-			NameTemplate:    jobName,
+			NameTemplate:    params.JobIDOrName,
+			Description:     params.JobIDOrName,
 			TargetLocaleIds: params.Locales,
 			Mode:            sdkjobs.ReuseExistingMode,
 			Salt:            sdkjobs.OrdinalSalt,
+			TimeZoneName:    "America/Los_Angeles",
```

Comment:
Is it possible to get laptop's time zone? Our `NameTemplate` allows some variables, and this timezone is realy improtant when we try to figure our the current date.
It is low priority, we can address it later.